Create Out-of-band resource load (HTTP).md (#241)
OWASP/www-community: pages/Out-of-band resource load (HTTP).md

Additionally, the application's processing of web content that is retrieved from arbitrary URLs exposes some important and non-conventional attack surface. An attacker can deploy a web server that returns malicious content, and then induce the application to retrieve and process that content. This processing might give rise to the types of input-based vulnerabilities that are normally found when unexpected input is submitted directly in requests to the application. The out-of-band attack surface that the application exposes should be thoroughly tested for these types of vulnerabilities.

> Attack Type: Server-Side

# Audit Guideline

> 1) Capture the base request of the vulnerable website in the burp community and send the request to the repeater.

Reviewer: ```suggestion
1) Capture the base request of the vulnerable website with a personal proxy and leverage that request via the proxy's resend functionality.
```

> 2) Change the request path to any internal or external URL. For eg. change the request path to https://www.attacker.com.

Reviewer: ```suggestion
2) Change the request path to any internal or external URL. For example change the request path to `https://www.attacker.com`.
```

> 3) The application will include the response of https://www.attacker.com with the response headers of vulnerable website.

Reviewer: ```suggestion
3) The application will include the response of `https://www.attacker.com` with the response headers of vulnerable website.
```

> with the response headers of vulnerable website.

It will?
Why is this relevant, it hasn't been mentioned till this point.

> 

Reviewer: ```suggestion
```
